Dockerfile

```diff
@@ -25,9 +25,9 @@ RUN export YARN_CACHE_FOLDER="$(mktemp -d)" \
 COPY --from=builder /app/dist /action-release/dist/
 RUN chmod +x /action-release/dist/index.js
 
-# move the sentry-cli binary to where the entrypoint expects it
-RUN mv /action-release/node_modules/@sentry/cli/sentry-cli /action-release/sentry-cli
-RUN chmod +x /action-release/sentry-cli
+# Workaround for this issue: https://github.com/libgit2/libgit2/issues/6663
+RUN printf '[safe]\n    directory = *\n' > /etc/gitconfig
```

Comment:
pedantically only the second line is a workaround -- the first line is "correct" for git here